### DISCUSSION TITLE
cannot load config file

### DISCUSSION THREAD

**Discussion by FuckingToasters** [any1/wayvnc#280] [Q&A]
hi, so i'm not sure which file ending the config file should have as this is not shown in the readme so i just created a file without any ending (no .json, .toml etc) which resulted in an issue thrown by GNuTLS.

```
GnuTLS: Failed to load credentials: Error while reading file.
ERROR: ../src/main.c: 932: Failed to enable TLS Authentication
```

Regards,
Infinimonster

**Comment by any1** [comment]
https://github.com/any1/wayvnc/blob/master/wayvnc.scd#configuration

There's no file name extension; it's just `config`.
